articles/aks/intro-kubernetes.md

```diff
@@ -53,7 +53,7 @@ To secure your AKS clusters, see [Integrate Azure AD with AKS][aks-aad].
 
 Container Insights has native integration with AKS, like collecting critical metrics and logs, alerting on identified issues, and providing visualization with workbooks or integration with Grafana. It can also collect Prometheus metrics and send them to [Azure Monitor managed service for Prometheus][azure-monitor-managed-prometheus], and all together deliver end-to-end observability.
 
-Logs from the AKS control plane components are collected separately in Azure as resource logs and sent to different locations, such as [Azure Monitor Logs][azure-monitor-logs]. For more information, see [Collect control plane logs][monitor-aks.md#collect-control-plane-logs].
+Logs from the AKS control plane components are collected separately in Azure as resource logs and sent to different locations, such as [Azure Monitor Logs][azure-monitor-logs]. For more information, see [Resource logs](monitor-aks.md#resource-logs).
```

Comment:
Line 178 specifies the URL to the article/section. And the URL should be - monitor-aks-reference.md#resource-logs

Thanks

Reply:
Hi @MGoedtel, Thanks for your feedback!
I understood the difference between the URL I previously committed.